modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -7,13 +7,21 @@
 [id="registry-configuring-registry-storage-swift-trust_{context}"]
 = Configuring the Image Registry Operator to trust Swift storage
 
-You must configure the Image Registry Operator to trust {rh-openstack-first} Swift storage.
+By disabling redirect, you can configure the Image Registry Operator to control whether clients such as {product-title} cluster builds or external systems like developer machines are redirected to pull images directly from {rh-openstack-first} Swift storage. This configuration is optional and depends on whether the clients trust the storage's SSL/TLS certificates.
 
+[NOTE]
+====
+In situations where clients to not trust the storage certificate, the `disableRedirect` option can be set to `true` to proxy traffic through the image registry. Consequently, however, the image registry might require more resources, especially network bandwidth, to handle the increased load. 
+
+Alternatively, if clients trust the storage certificate, the registry can allow redirects. This reduces resource demand on the registry itself. 
+
+Some users might prefer to configure their clients to trust their self-signed certificate authorities (CAs) instead of disabling redirects. If you are using a self-signed CA, you must decide between trusting the custom CAs or disabling redirects.
+====
 // to allow the client to pull the image layers from the image registry rather than from links directly from Swift.
 
 .Procedure
 
-* From a command line, enter the following command to change the value of the `spec.disableRedirect` field in the `config.imageregistry` object to `true`:
+* From a command line, enter the following command to change the value of the `spec.disableRedirect` field in the `config.imageregistry` object to `true`. This ensures that the image registry will proxy traffic instead of relying on Swift storage.
```

Comment:
```suggestion
* To ensures that the image registry proxies traffic instead of relying on Swift storage, change the value of the `spec.disableRedirect` field in the `config.imageregistry` object to `true` by running the following command:
```